src/inference/src/compiled_model.cpp

```diff
@@ -22,6 +35,7 @@ namespace ov {
 
 CompiledModel::~CompiledModel() {
     _impl = {};
+    release_memory_to_os();
```

Comment:
Do you have any measurements of rss and performance before and with this change?

Reply:
see description

Reply:
Yes my fault.